server/routes/message.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const router = express.Router();
+
+const auth = require("../middleware/auth");
+
+// Connect to conversation controller
+const messageController = require("../controllers/message");
+
+/* POST route to add a message - POST MESSAGE operation. */
```

Comment:
As with the convo routes, I would also remove all these comments above the routes as the code will explain itself.